similarity-data-api/src/main/java/org/phenotips/data/similarity/Exome.java

```diff
@@ -19,23 +19,24 @@
 
 import org.xwiki.stability.Unstable;
 
+import java.util.List;
 import java.util.Set;
 
 import net.sf.json.JSONArray;
 
 /**
- * A basic representation of a genotype of a patient, with scored Variant objects in scored genes.
+ * A basic representation of a exome of a patient, with scored {@link Variant} objects in scored genes.
  *
  * @version $Id$
- * @since 1.0M8
+ * @since
  */
 @Unstable
-public interface Genotype
+public interface Exome
 {
     /**
      * Get the names of all genes with variants in the patient.
      *
-     * @return an unmodifiable set of gene names with variants in the patient
```

Comment:
Because when I was going through it I decided that it didn't seem necessary to have the unmodifiable constraint specified within the interface. It felt like noise?